integrationtests/src/test/java/io/kroxylicious/proxy/filter/ApiVersionsMarkingFilter.java

```diff
@@ -39,4 +41,21 @@ public CompletionStage<RequestFilterResult> onRequest(ApiKeys apiKey, RequestHea
             return context.forwardRequest(header, request);
         });
     }
+
+    public static class Contributor implements FilterContributor {
+        @Override
+        public String getTypeName() {
+            return "ApiVersionsMarkingFilter";
```

Comment:
Tom suggested we drop the idea of aliases in favour of using the full classname or the short name as an alias. So we would be able to tolerate two forms in the config YAML:

1. io.kroxylicious.proxy.filter.ApiVersionsMarkingFilter
2. ApiVersionsMarkingFilter

Reply:
That might be easier to explain to the users, rather than having to describe custom aliases.

Reply:
+1 to the idea.

When trying to resolve a shortname, would we scan the classpath looking for a match?  We'd need to deal with the possibility of multiple matches, or matches that didn't implement the right interface.

Reply:
One way would be Contributor changing to something like:

```
@NonNull
Class<? extends T> getType();
```
where it returns the concrete type of the Filter, then when we are iterating over contributors we would compare the String from configuration to the class.

Lets defer this to a second PR

Reply:
How would that play with the idea of splitting up the multi tenant filter? Thats the use case when aliases are ideal right?

Reply:
With the MT filter, I'd imagined that MultiTenantFilter would be a CompositeFilter, then n private filters within it.   In my mental model it is only the MultiTenantFilter that would have the Contributor. Right?

Reply:
There was another thread about this, the question for me has become why is `CompositeFilter` a thing, shouldn't Contributor return a set of filters?